tests/utils/date_parser_tests.py

```diff
@@ -291,3 +295,6 @@ def test_DateRangeMigration(self):
 
         field = "   8 days   "
         self.assertRegex(field, DateRangeMigration.x_dateunit)
+
+        field = "last week"
+        self.assertNotRegex(field, DateRangeMigration.x_dateunit)
```

Comment:
One nit, add a test case :
```suggestion

        field = "last week"
        self.assertNotRegex(field, DateRangeMigration.x_dateunit)
        
        field = "1 years ago"
        self.assertNotRegex(field, DateRangeMigration.x_dateunit)
```